### PR TITLE
Add Doc: Use Jenkins Shared Libraries in a Pipeline

### LINKED ISSUE #1430: [Have video] Add a tutorial to introduce how to use Jenkins sharedlibrary in KuebSphere Devops
Many users choose to adopt [shared-library](https://www.jenkins.io/doc/book/pipeline/shared-libraries/) to improve their Pipeline practice. But some KubeSphere users might curious about can I use it in KubeSphere DevOps? The answer is definitely yes! But how?

So, in my opinion, providing a tutorial is meaningful. If anyone wants to help to write it, I've [some videos (in Chinese)](https://www.bilibili.com/video/BV1fp4y1r7Dd?p=20) about the shared library. Hopefully, those videos can give you some hints.

/area devops
/priority medium

### PR DESCRIPTION
**Pull request by Felixnoo**
Signed-off-by: Felixnoo <felixliu@yunify.com>

Fix #1430

### CODE REVIEW

File: content/en/docs/devops-user-guide/how-to-use/jenkins-shared-library.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+4. After a while, the pipeline will run successfully.
+
+   ![run-successfully](/images/docs/devops-user-guide/using-devops/jenkins-shared-library/run-successfully.png)
+
+5. You can click the **Success** record under **Status**, and then click **Show Logs** to view the log details.
+
+   ![log-details](/images/docs/devops-user-guide/using-devops/jenkins-shared-library/log-details.png)
+
+6. You can also use a Jenkinsfile starting with `@Library('demo-shared-library') _`  ???
```
Comment (Felixnoo) [inline content/en/docs/devops-user-guide/how-to-use/jenkins-shared-library.md:116]: Is it necessary to demonstrate the method of using the shared library through a Jenkinsfile starting with `@Library('<the shared library name set in Jenkins>') _`?
As I understand it, this method is different from the method already demonstrated in the doc.

If so, I need someone to provide the Jenkinsfile...

**Comment by Felixnoo** [comment]
/assign @LinuxSuRen @JohnNiang 
Hi, please take a look at this PR and my comment when available.  [Preview link](https://deploy-preview-1731--kubesphere-v3-1.netlify.app/docs/devops-user-guide/how-to-use/jenkins-shared-library/). Thx.

**Comment by Felixnoo** [comment]
Fix #1430

**Comment by JohnNiang** [comment]
> Fix #1430

Please put `Fix #1430` in PR description.

**Comment by Felixnoo** [comment]
/assign @Patrick-LuoYu 
Please review this PR, thx.

**Pull request merged**